roadway_objects/src/roadway_objects_worker.cpp

```diff
@@ -1,56 +1,61 @@
-/*
- * Copyright (C) 2019-2022 LEIDOS.
- *
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not
- * use this file except in compliance with the License. You may obtain a copy of
- * the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
- * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
- * License for the specific language governing permissions and limitations under
- * the License.
- */
+// Copyright 2023 Leidos
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
 #include "roadway_objects/roadway_objects_worker.hpp"
 
 namespace roadway_objects
 {
-RoadwayObjectsWorker::RoadwayObjectsWorker(carma_wm::WorldModelConstPtr wm, const PublishObstaclesCallback& obj_pub, rclcpp::node_interfaces::NodeLoggingInterface::SharedPtr logger)
-  : obj_pub_(obj_pub), wm_(wm), logger_(logger)
+RoadwayObjectsWorker::RoadwayObjectsWorker(
+  carma_wm::WorldModelConstPtr wm, const PublishObstaclesCallback & obj_pub,
+  rclcpp::node_interfaces::NodeLoggingInterface::SharedPtr logger)
+: obj_pub_(obj_pub), wm_(wm), logger_(logger)
 {
 }
 
-void RoadwayObjectsWorker::externalObjectsCallback(const carma_perception_msgs::msg::ExternalObjectList::UniquePtr obj_array)
+void RoadwayObjectsWorker::externalObjectsCallback(
+  const carma_perception_msgs::msg::ExternalObjectList::UniquePtr obj_array)
 {
   carma_perception_msgs::msg::RoadwayObstacleList obstacle_list;
   auto map = wm_->getMap();
-  if (!map)
-  {
-    RCLCPP_WARN_STREAM(logger_->get_logger(), "roadway_objects could not process external objects as no semantic map was available");
+  if (!map) {
+    RCLCPP_WARN_STREAM(
+      logger_->get_logger(),
+      "roadway_objects could not process external objects as no semantic map was available");
     return;
   }
 
-  if (map->laneletLayer.size() == 0)
-  {
-    RCLCPP_WARN_STREAM(logger_->get_logger(), "roadway_objects could not process external objects as the semantic map does not contain any lanelets");
+  if (map->laneletLayer.size() == 0) {
```

Comment:
Since we are refactoring, can we change this check to .empty()?

Reply:
The `RoadwayObjectsWorker` is going to be removed when #2144 is resolved, so I'm inclined to keep that line as is. What are your thoughts?